### PR TITLE
Adding MigratingSessionHandler docs

### PR DESCRIPTION
**Pull request by rossmotley**
Adding `MigratingSessionHandler` documentation for https://github.com/symfony/symfony/pull/26096

### CODE REVIEW

File: components/http_foundation/session_configuration.rst [hunk 1]
```diff
@@ -87,6 +88,35 @@ Example usage::
...
+
+The :class:`Symfony\\Component\\HttpFoundation\\Session\\Storage\\Handler\\MigratingSessionHandler`
+can be used to migrate between old and new save handlers without losing session data.
+
+It can be used to support the following workflow:
+
+* Switch to the migrating handler, with your new handler as the write-only one. The old handler behaves as usual and sessions get written to the new one.
+* After your session gc period, verify the data in the new handler
+* Update the migrating handler to use the old handler as the write-only one, so the sessions will now be read from the new handler. This step allows easier rollbacks.
+* After verifying everything, switch from the migrating handler to the new handler
```
Comment (iltar) [inline components/http_foundation/session_configuration.rst]: Missing a full stop at the end

` * After verifying everything, switch from the migrating handler to the new handler.`

**Review by linaori — CHANGES_REQUESTED** [review]

**Review by linaori — APPROVED** [review]

**Review by ndench — APPROVED** [review]

**Comment by xabbuh** [comment]
Would it make sense to move the detailed description to a new guide and only link to it from this page?

**Comment by iltar** [comment]
@xabbuh it looks like the page is becoming quite big already with this addition, it might be an option

**Review by javiereguiluz — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks @rossmotley. This was a really nice first contribution to Symfony Docs! Thank you.

**Pull request merged**